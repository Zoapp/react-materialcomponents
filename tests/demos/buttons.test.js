import React from "react";
import renderer from "react-test-renderer";
import Buttons from "demos/components/buttons";

describe("demos/components/buttons", () => {
  it("renders itself correctly", () => {
    const component = renderer.create(<Buttons />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
