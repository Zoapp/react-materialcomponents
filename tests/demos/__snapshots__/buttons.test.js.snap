// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`demos/components/buttons renders itself correctly 1`] = `
<section>
  <h1>
    Button examples
  </h1>
  <div>
    <div
      style={
        Object {
          "padding": "16px",
        }
      }
    >
      <button
        className="mdc-button"
        disabled={false}
        id={null}
        onClick={null}
        type="button"
      >
        
        Hello
      </button>
    </div>
    <div
      style={
        Object {
          "padding": "16px",
        }
      }
    >
      <button
        className="mdc-button mdc-button--raised"
        disabled={false}
        id={null}
        onClick={null}
        type="button"
      >
        <i
          className="mdc-button__icon material-icons"
        >
          favorite
        </i>
        Hello
      </button>
    </div>
    <div
      style={
        Object {
          "padding": "16px",
        }
      }
    >
      <button
        className="mdc-button mdc-button--raised"
        disabled={false}
        id={null}
        onClick={null}
        type="button"
      >
        
        Hello
      </button>
    </div>
    <div
      style={
        Object {
          "padding": "16px",
        }
      }
    >
      <button
        className="mdc-button mdc-button--raised"
        disabled={true}
        id={null}
        onClick={null}
        type="button"
      >
        
        Hello
      </button>
    </div>
    <div
      style={
        Object {
          "padding": "16px",
        }
      }
    >
      <button
        className="mdc-button mdc-button--unelevated mdc-button--compact secondary-filled-button"
        disabled={false}
        id={null}
        onClick={null}
        type="button"
      >
        
        Hello
      </button>
    </div>
    <div
      style={
        Object {
          "padding": "16px",
        }
      }
    >
      <button
        className="mdc-button mdc-button--stroked mdc-button--dense"
        disabled={false}
        id={null}
        onClick={null}
        type="button"
      >
        
        Hello
      </button>
    </div>
    <div
      style={
        Object {
          "padding": "16px",
        }
      }
    >
      <a
        className="mdc-button mdc-button--raised"
        href="#"
        id={null}
      >
        
        Hello
      </a>
    </div>
  </div>
  <h2>
    Properties:
  </h2>
</section>
`;
